test/unity/toolkits/drawing_classifier/test_dc_data_iterator.cxx

```diff
@@ -14,63 +14,13 @@
 #include <model_server/lib/image_util.hpp>
 #include <core/util/test_macros.hpp>
 
+#include "data_utils.hpp"
```

Comment:
again, it's better to split into 2 prs. a lot of those are related to data_iterator, which I heavily depend on both in development and test.